src/OpenLoco/src/Graphics/Gfx.cpp

```diff
@@ -81,11 +81,14 @@ namespace OpenLoco::Gfx
 
         if (header.numEntries != G1ExpectedCount::kDisc)
         {
-            Logging::warn("G1 element count doesn't match expected value:\nExpected {}; Got {}", G1ExpectedCount::kDisc, header.numEntries);
             if (header.numEntries == G1ExpectedCount::kSteam)
             {
                 Logging::info("Got Steam G1.DAT variant, will fix elements automatically.");
             }
+            else
+            {
+                Logging::verbose("G1 element count doesn't match expected value:\nExpected {}; Got {}", G1ExpectedCount::kDisc, header.numEntries);
```

Comment:
I think a warning would be better here since you moved it to a condition to where we don't know where the G1 is from.